base/custom_metrics.go

```diff
@@ -71,42 +72,45 @@ type Params struct {
 	Value string `json:"value" yaml:"value"`
 }
 
-// collectDatabaseInputs is the input to the collect-metrics-database task
-type collectDatabaseInputs struct {
-	// Providers is the set of labels/names of the data sources
-	Providers []string `json:"providers" yaml:"providers"`
+// customMetricsInputs is the input to the custommetrics task
```

Comment:
We want to call this task `custommetrics` which is what it really is (as opposed to built-in metrics).